client/src/data/configs.js

```diff
@@ -0,0 +1,24 @@
+const configs = [
+    {
+      name: "Config 1",
+      description: "This is the first config",
+      id: 1,
+    },
+    {
+      name: "Config 2",
+      description: "This is the second config",
+      id: 2,
+    },
+    {
+      name: "Config 3",
+      description: "This is the third config",
+      id: 3,
+    },
+    {
+      name: "Config 4",
+      description: "This is the fourth config",
+      id: 4,
+    },
+  ];
+
+export default configs;
```

Comment:
I see a github linter-related warning (I didn't check yet if there's a js-specific linter in the project yet :) ) on this file, loadClients.js, and loadTests.js saying there's no new line at the end of the file. 

What I'm seeing:
![image](https://user-images.githubusercontent.com/3301452/127902195-c8d79ca4-2abb-4508-9455-26c32c4d7850.png)

Reply:
We do have eslint installed on the project, but I am not sure why it did not flag that. I will go in and change it!

Reply:
It's probably not a default setting with eslint :).  This is one that I've mainly seen popping up on github PR's.